sql/core/src/test/resources/sql-tests/inputs/pipe-operators.sql

```diff
@@ -74,6 +74,10 @@ create temporary view windowTestData as select * from values
 -- SELECT operators: positive tests.
 ---------------------------------------
 
+-- Selecting from a constant.
+from t
```

Comment:
As I understand correctly, this PR also allows supporting "FROM t" outside the context of SQL pipe operators, as a synonym for "TABLE t"? If so, that satisfies the requirements for SQL pipe operators, maybe we should have a test that shows "FROM t" by itself as well in this to show that it works.

Reply:
Added test cases with FROM and TABLE alone.